themes-default/slim/package.json

```diff
@@ -110,23 +113,72 @@
       "browser"
     ],
     "globals": [
-      "PNotify",
-      "LazyLoad",
-      "_",
-      "log",
-      "s",
-      "$",
-      "MEDUSA",
-      "api",
-      "apiv1",
-      "apiRoute",
-      "jQuery",
-      "axios",
-      "webRoot",
-      "apiRoot",
-      "apiKey",
-      "Vuex",
-      "dateFns"
+        "PNotify",
+        "LazyLoad",
+        "_",
+        "log",
+        "s",
+        "$",
+        "MEDUSA",
+        "jQuery",
+        "axios",
+        "dateFns"
+    ],
+    "overrides": [
+      {
+        "files": "src/**/*",
+        "plugins": [
+          "jquery"
+        ],
+        "rules": {
```

Comment:
I wonder if you can do the following instead of listing all the rules manually:

Keep the "plugins" override, remove "rules" override.
Add to "extends" above: `jquery/deprecated` ([has the same rules](https://github.com/dgraham/eslint-plugin-jquery/blob/v1.3.1/index.js#L52))
(I'm not sure if the overrides are merged with the "root" XO config values)

Reply:
The overrides replaces the root xo config.